### PR TITLE
docs: add committing style guide and fix typos (#244)

### PR DESCRIPTION
**Pull request by WaltherTrgovac**
I have added the link for the committing style guide and fixed some typos.

### CODE REVIEW

**Comment by tstorek** [comment]
@WaltherTrgovac I just recognized  that you changed, e.g. `devive` to 'device' in the example. This actually on perpose to highlight it when converting to markdown.

**Comment by tstorek** [comment]
> I see what you mean, but .md and .py are two separate files, so the .md file stays unchanged. But if you want, I can revert the changes. Besides that, my idea was to make all the #ToDo comment sections and therefore .md files written in full sentences. And other comments I would leave as they are.

That's totally correct. However, we used to have a parser to generate the *.md from *.py. Not sure if it still exists ;)

**Comment by WaltherTrgovac** [comment]
I assumed that so I think it is better to leave them as it was, I will revert the changes.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -57,23 +59,23 @@ def foo(dummy: str , dummy2: Union[str, int]):
...
-- Test as many things as possible. Even seemingly silly tests like correct input-format help prevent future problems for new users
-- use the `self.assertSOMETHING` functions provided by `unittest`. This way a test failure is presented correctly An error inside your test function will not be handeled as a failure but an error.
+- Test as many things as possible. Even seemingly silly tests like correct input-format help prevent future problems for new users.
+- Use the `self.assertSOMETHING` functions provided by `unittest`. This way a test failure is presented correctly. An error inside your test function will not be handled as a failure but as an error.
 - If the success of your test depends on the used device, you can use decorators like `skip()`, `skipif(numpy.__version__<(1, 0), "not supported with your numpy version")`, etc. 
 - `setUp()` and `tearDown()` are called before and after each test. Use this functions to define parameters used in every test, or to close applications like Dymola once a test is completed.
 - See the [unittest-documentation](https://docs.python.org/3/library/unittest.html#organizing-tests) for further information
```
Comment (Maghnie) [inline CONTRIBUTING.md:75]:
```suggestion
- See the [unittest-documentation](https://docs.python.org/3/library/unittest.html#organizing-tests) for further information.
```

**Review by Maghnie — CHANGES_REQUESTED** [review]
Thanks for the updates!

**Review by djs0109 — APPROVED** [review]

**Comment by djs0109** [comment]
@Maghnie It looks good to me. You can merge if it also looks good for you now :)

**Review by Maghnie — APPROVED** [review]

**Pull request merged**